templates/home/index.html

```diff
@@ -438,10 +438,10 @@ <h1 data-aos="fade-up">Contact Us</h1>
         </p>
         <div>
             <div class="con-box1" data-aos="fade-down-right">
-                <p><i class="fas fa-mobile fa-2x"></i> +91-xxxxx-xxxxx</p>
+                <p><i style="padding-right: 5px;" class="fas fa-phone fa-rotate-90 fa-2x"></i><a style="color: #000; a:hover: black" href="tel:+91-xxxxx-xxxxx" onMouseOver="this.style.color='#ff8c00'"onMouseOut="this.style.color='#000'"> +91-xxxxx-xxxxx </a></p>
             </div>
             <div class="con-box1" data-aos="fade-down-left">
-                <p><i class="fas fa-envelope fa-2x"></i> jagrati@iiitdmj.ac.in</p>
+                <p><i style="padding-right: 5px;" class="fas fa-envelope fa-2x"></i><a style="color: #000;" href="mailto:jagrati@iiitdmj.ac.in" onMouseOver="this.style.color='#ff8c00'"onMouseOut="this.style.color='#000'"> jagrati@iiitdmj.ac.in </a></p>
```

Comment:
Instead of writing everything here, use external CSS. Also, the email is still not centered w.r.t. the icon.

![image](https://user-images.githubusercontent.com/39924567/119492660-f5101b80-bd7c-11eb-8835-66348ccbf52d.png)

One more thing is that the icon should also show the same effect on hover (you can put both the icon and the email/phone number inside the `a` tag).